src/RFC-0181_BulletproofsPlus.md

```diff
@@ -253,6 +253,12 @@ This means single-proof verification has $b = 1$.
 
 Verification in Bulletproofs+ is slightly faster than in Bulletproofs.
 
+### Proving efficiency
+
+It is more challenging to compare proving efficiency, since generation of a range proof does not reduce cleanly to a single multiscalar multiplication evaluation for either Bulletproofs or Bulletproofs+ range proofs.
+However, we note that the overall complexity between the inner-product arguments in the proving systems is similar in terms of group operations; outside of these inner-product arguments, Bulletproofs+ requires fewer group operations.
+Overall efficiency is likely to depend on specific optimizations.
+
```

Comment:
```suggestion

Initial benchmarks to create a single proof up to 32 aggregated proofs indicated that Bulletproofs+ is 30% more expensive on average than Bulletproofs.
```

Reply:
My personal preference would be to defer these to the implementation doucmentation, which is where we already have a rough estimate for verification. This makes it easier to keep the data relevant through changes, while ensuring that the RFC contains theoretical results that aren't strictly dependent on implementation.

Reply:
Added in a new commit.